packages/@o3r/core/schematics/rule-factories/o3r-basic-updates/index.ts

```diff
@@ -9,7 +9,7 @@ import { ngAddPackages, readAngularJson, readPackageJson } from '@o3r/schematics
  * @param o3rCoreVersion
  * @param projectType
  */
-export function o3rBasicUpdates(pName: string | null, o3rCoreVersion?: string, projectType?: 'application' | 'library'): Rule {
+export function o3rBasicUpdates(pName: string | null, o3rCoreVersion?: string, projectType?: 'application' | 'library' | 'monorepo'): Rule {
```

Comment:
Why not considering `undefined` as monorepo case, as the type is mandatory otherwise?